handlers/additem.js

```diff
@@ -5,6 +5,9 @@ const {InlineKeyboard} = require('node-telegram-keyboard-wrapper');
 const sprintf = require("sprintf-js").sprintf;
 const COMMAND_ID = commands.indexOf('additem');
 const MOD_COMMAND_ID = commands.indexOf('addmod');
+
+const REMARK_TOO_LONG_RESPONSE = "Remark is longer than 60 characters, try again with a shorter remark!";
```

Comment:
You can leave this const in this file. Do construct it at runtime using the max character value!

Reply:
Done, I constructed it by concatenating strings, would it better to use the util.format function?